hacktoberfest_challenges/datasets_without_language.md

```diff
@@ -0,0 +1,105 @@
+```
+
+Having this field filled in is essential for users to find datasets in their language and give a better idea of the languages that the Hub covers. However, the dataset's author has only sometimes filled this field. This challenge is to fill in the `language` field for datasets that don't have it filled in.
+
+
+## How to contribute?
+
+How can you help improve the coverage of language metadata on the Hub?
+
+For each dataset, the workflow is the following:
+
+1. Find a dataset that doesn't have the `language` field filled in. You can find a list of datasets without the `language` field filled in [here](#datasets-without-language-field-filled-in). We start with datasets that have the most downloads and likes.
+2. Check that the dataset doesn't already have a PR to add a language tag(s). Someone else may have already started working on it. You can check this by looking in the discussion section of the dataset page. 
+3. If there is no PR to add language metadata already open, your next step is to identify the language (if possible for the dataset). There are a few main ways you can often identify the language
+   1. The dataset's name. Often, the name of the dataset will include the language. Sometimes as a full name, i.e. `imdb_german` or sometimes as a language code, i.e. `imdb_de`. You can use that as the language tag if the dataset name includes the language.
+   2. The dataset card will sometimes mention the language(s) of the dataset explicitly. 
+   3. Many datasets will have an active [dataset viewer](https://huggingface.co/docs/hub/datasets-viewer) for the dataset. This will allow you to see examples from the dataset. You may identify the language by looking at the text examples. 
+   4. Sometimes, the dataset will have a column specifying the language of the text. You can use this column to fill in the language tag(s).
+   5. If the dataset viewer is available for the dataset, but you don't recognize the language, you can use the [facebook/fasttext-language-identification](https://huggingface.co/facebook/fasttext-language-identification) model or [Google Translate](https://translate.google.com/) to try to identify the language. 
+4. Once you've identified the language(s) of the dataset, you can add the language tag(s) to the dataset card. You can do this by clicking the `Edit` button on the dataset card. This will open a PR to the dataset repo. You can add the language tag(s) to the `language` field in the dataset card.
+5. Once done, open a PR on GitHub to update the table below. Once merged, this will count as a Hacktoberfest contribution! Add the `pr_url` (the one on the Hub) and a status (opened, merged, closed) in the PR. 
+6. Adding a language tag to some of the datasets below may not make sense. If so, add `not relevant` as the link in the `pr_url`. There may also be datasets where you need help with the language. In these cases, you can open a discussion to suggest a language tag(s) is added to the dataset. 
+
+## F.A.Q.
```

Comment:
Another question I had: what about multiple-language datasets? is it worth adding a few languages even if I don't know if the list is exhaustive?